airflow/providers/databricks/operators/databricks.py

```diff
@@ -287,8 +287,8 @@ def __init__(
             self.json["git_source"] = git_source
         if access_control_list is not None:
             self.json["access_control_list"] = access_control_list
-
-        self.json = normalise_json_content(self.json)
+        if self.json:
```

Comment:
As `self.json`, is being set above lines, it can't be null.

Reply:
True. It `escapes` the init checks. Using the `normalize_json_content` function in the execute function as documented raises errors.

Reply:
Well. It can be empty dict after all the if's, so it saves a little to check instances in `normalize_json_content` so leaving it is a small optimisation.

Reply:
An empty dict checks the same as a null value.